src/index.ts

```diff
@@ -1,15 +1,22 @@
 import express, { Application, Request, Response } from 'express';
+import userRoutes from './routes/userRoutes';
+import todoRoutes from './routes/todoRoutes';
+import { errorHandler } from './middlewares/errorHandler';
+import connectDB from './database';
```

Comment:
it would be better to have this as:
`import connectToDB from '.services/databaseService.ts'`